t/plugin/forward-auth.t

```diff
@@ -206,6 +206,42 @@ property "request_method" validation failed: matches none of the enum values
                         "uri": "/ping"
                     }]],
                 },
+                {
+                    url = "/apisix/admin/routes/4",
+                    data = [[{
+                        "plugins": {
+                            "forward-auth": {
+                                "uri": "http://127.0.0.1:1964/auth",
```

Comment:
Using the nonexistent port is very unstable, could you try to use the  `serverless-pre-function` plugin and `core.response.exit(444)` to mock the unavailable auth server